### PR TITLE
A link to the mattermost added

### PR DESCRIPTION
**Pull request by vlisovsk**
Think this is a good alternative to the mailing lists, especially for the beginners.

### CODE REVIEW

File: first-analysis-steps/asking-questions.md [hunk 1]
```diff
@@ -17,13 +17,15 @@ For LHCb specific questions your best bet are the LHCb mailing
 lists like: 
 [lhcb-distributed-analysis@cern.ch](mailto:lhcb-distributed-analysis@cern.ch),
 [lhcb-davinci@cern.ch](mailto:lhcb-davinci@cern.ch),
-and [lhcb-soft-talk@cern.ch](mailto:lhcb-soft-talk@cern.ch). For
-more general questions [Stack Overflow](http://stackoverflow.com/) and
+and [lhcb-soft-talk@cern.ch](mailto:lhcb-soft-talk@cern.ch). 
+A recent alternative to the mailing lists is the [Mattermost chat](https://mattermost.web.cern.ch/lhcb) having dedicated channels ('davinci', 'Distributed Analysis' etc). It is a great place to ask your question if you feel not confident enough to write a formal email into the mailing list. 
+
+For more general questions [Stack Overflow](http://stackoverflow.com/) and
```
Comment (alexpearce) [inline first-analysis-steps/asking-questions.md]: * "if you feel not confident.." -> "if you're not confident.."
* "into the mailing list" -> "to the mailing list"
* "mailing list." -> "mailing list, or you'd just prefer a more informal setting."

Comment (vlisovsk) [reply first-analysis-steps/asking-questions.md]: Sorry for my English

Comment (alexpearce) [reply first-analysis-steps/asking-questions.md]: Your English is great! 🙇  We all make mistakes.

**Comment by alexpearce** [comment]
Thank you! 🍻

**Pull request merged**